src/app/code-block/code-block.component.ts

```diff
@@ -66,9 +68,11 @@ export class CodeBlockComponent implements OnDestroy {
   ngOnDestroy() {
   }
 
-  focus() {
+  focus($event?: any) {
```

Comment:
I believe we have been trying to use JQuery.TriggeredEvent for the event type

Reply:
@deep7102 You will want something like this >
focus($event?: JQuery.TriggeredEvent) {